File: abc/abc275/a.py
# https://atcoder.jp/contests/abc275/submissions/36082888

#whileを使う解法(1)

#入力
n = int(input())
h = list(map(int, input().split()))

#リストを探索
ans = 0;
ma = 0;
i = 0;

while i < n:
  if h[i] > ma:
    ans = i + 1;
    ma = h[i];
  i += 1

#出力
print(ans);

File: abc/abc275/a.cs
// https://atcoder.jp/contests/abc275/submissions/46938006

class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    string[] h = Console.ReadLine().Split(" ");

    int idx = 0;
    for(int i = 0; i < n; i++)
    {
      if(int.Parse(h[idx]) < int.Parse(h[i]))
      {
        idx = i;
      }
    }

    Console.Write(idx + 1);
  }
}

